CONTRIBUTING.md

```diff
@@ -1,21 +1,49 @@
 # Contributing to Fingerprint Pro Server API OpenAPI
 
-The repository contains [Fingerprint Pro Server API](https://dev.fingerprint.com/docs/server-api) definition in OpenAPI format.
+The repository contains OpenAPI definition files for various [Fingerprint Pro Server-side APIs](https://dev.fingerprint.com/reference).
 
-## Schema
+## Schemas
 
-You can find the definition in the [schemas](/schemas) folder. You can find information about OpenAPI 3.0.3 in the [official specification](https://spec.openapis.org/oas/v3.0.3).
+You can find the schemas definitions in the [schemas](/schemas) folder. You can find information about OpenAPI 3.0.3 in the [official specification](https://spec.openapis.org/oas/v3.0.3).
 
-## Demo app
+## Contributing to the schema definition
 
-Code of demo application can be found in [src](src) folder. It uses [Swagger UI](https://github.com/swagger-api/swagger-ui) to render schema and provide sandbox.
+- If you are not familiar with the OpenAPI format, you can watch this [video summary](https://www.youtube.com/watch?v=pRS9LRBgjYg) (1.5x speed recommended).
+- Authoring schemas is easier with IDE support. For VS Code, you can use [OpenAPI (Swagger) Editor](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi). You can look for similar extensions for your IDE of choice.
+- OpenAPI schema is repetitive, the easiest way to add something (schema, endpoint, response, property...) is to copy-paste an existing entity and modify it. Try to follow the established patterns for naming, ordering, example values, example files, etc.
+- You can reference the [OpenAPI specification](https://spec.openapis.org/oas/v3.0.3) for more information.
+- If you are adding a new signal, please also add it to `get_event_200.json`, `webhook.json` and other relevant files in the `examples` folder.
```

Comment:
Let's also mention [get_event_200_all_errors.json](https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/blob/main/examples/get_event_200_all_errors.json) example too, because people always forgot about it.

Reply:
Fixed, thank you https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/110/commits/4c5e2edcd2789fc0b83d249bce611fc3c643636b